### PR TITLE
create as matrix in numpy frontend and associated test

### LINKED ISSUE #13320: asmatrix
asmatrix for NumPy frontend add manipulation routines

### PR DESCRIPTION
**Pull request by younggraham-t**
Close #13320

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_kind_of_array.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+def test_numpy_asmatrix(
+    dtypes_and_x,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    input_dtypes, x = dtypes_and_x
+    
+    np_frontend_helpers.test_frontend_function(
```
Comment (kurshakuz) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_kind_of_array.py:25]: `np_frontend_helpers` is not defined.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_kind_of_array.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        input_dtypes=input_dtypes,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        
+        out=None,
+        dtype=dtype,
```
Comment (kurshakuz) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_kind_of_array.py:33]: `dtype` is not defined.

File: ivy/functional/frontends/numpy/manipulation_routines/changing_kind_of_array.py [hunk 3]
```diff
@@ -1 +1,15 @@
 # local
+
+import ivy
+
+from ivy.functional.frontends.numpy.func_wrapper import inputs_to_ivy_arrays
+from ivy.functional.frontends.numpy import matrix
```
Comment (kurshakuz) [inline ivy/functional/frontends/numpy/manipulation_routines/changing_kind_of_array.py:6]: `matrix` is unused.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**